### ISSUE TITLE
[Bug]: Download In Android App Does Not Work

### ISSUE THREAD

**Issue by weswitt** [advplyr/audiobookshelf#1259]
### Describe the issue

I cannot download books in the Android app. Using the browser the downloads work just fine.  In the Android app I click on the download button and it does nothing. The progress just stays at 0%.  This could certainly be something I did with the config but i don't see it. 

### Steps to reproduce the issue

1. Try to download a book

### Audiobookshelf version

2.2.5

### How are you running audiobookshelf?

Debian/PPA

**Comment by weswitt** [comment]
also of note is that streaming works.

i just noticed that eventually the i got a notification that the download finished but could no create library item.

**Comment by hobesman** [comment]
We're you able to set a folder for downloading to? And it said something to the effect of "Access granted" for that folder?

**Comment by weswitt** [comment]
yes. in fact i tried creating a new directory on my phone and setting ab to use it. i got the notification that the permission was set, but downloading seems to do nothing.

i can see in the ab server log that the phone is connecting:

Dec 09 09:50:17 audiobookshelf audiobookshelf[595]: [2022-12-09 09:50:17]  INFO: [Server] Socket Connected Ou3U9_djYaJ4HS6qAAAZ
